src/lib/y2network/widgets/gateway.rb

```diff
@@ -21,8 +21,12 @@ def label
       end
 
       def help
-        # TODO: original also does not have help
-        ""
+        _(
+          "<p><b>Gateway</b> defines gateway for given route. Special value here is '-' " \
```

Comment:
defines the IP address of a host which routes the packets to a remote host or network. It can be blank for rejection or device routes.